src/icalendar/prop/__init__.py

```diff
@@ -810,51 +810,74 @@ def from_ical(ical):
 class vDatetime(TimeBase):
     """Date-Time
 
-    Value Name:  DATE-TIME
+    Value Name:
+        DATE-TIME
+
+    Purpose:
+        This value type is used to identify values that specify a
+        precise calendar date and time of day. The format is based on
+        the ISO.8601.2004 complete representation.
+
+    Format Definition:
+        This value type is defined by the following notation:
+
+        .. code-block:: text
+
+            date-time  = date "T" time
+
+            date       = date-value
+            time       = time-hour time-minute time-second [time-utc]
+
+            date-value = date-fullyear date-month date-mday
+            time-hour  = 2DIGIT        ;00-23
+            time-minute = 2DIGIT       ;00-59
+            time-second = 2DIGIT       ;00-60
+            time-utc   = "Z"
```

Comment:
This is a good addition, but it needs more date stuff. Also it's better to group them by date and time and vertically align the `=` and `;`.

```suggestion
            date-time  = date "T" time

            date       = date-value
            date-value         = date-fullyear date-month date-mday
            date-fullyear      = 4DIGIT
            date-month         = 2DIGIT        ;01-12
            date-mday          = 2DIGIT        ;01-28, 01-29, 01-30, 01-31
                                               ;based on month/year
            time               = time-hour time-minute time-second [time-utc]
            time-hour          = 2DIGIT        ;00-23
            time-minute        = 2DIGIT        ;00-59
            time-second        = 2DIGIT        ;00-60
            time-utc           = "Z"
```